monitor/dbt_project/macros/get_new_alerts_where_clause.sql

```diff
@@ -10,7 +10,7 @@
     {%- set row_count = elementary.get_row_count(alerts_model_relation) %}
     {%- if row_count > 0 %}
         {% set where_clause %}
-            where detected_at > {{ last_alert_time_with_backfill }}
+            detected_at > {{ last_alert_time_with_backfill }}
```

Comment:
also instead of the inner subquery - why don't we just check if detected_at > {{ last_alert_time_with_backfill }} and alert_sent = false?

Reply:
Got it

Reply:
@IDoneShaveIt Is there a good reason to have the `alert_sent = true` here?

Reply:
Yeah, because if you won't filter those alerts that already sent, the merging will override the existing sent ones, and will create them with false.
This will cause us sending the same alerts multiple times.

Merging in incremental delete the exist row, and write the new one (on redshift when they don't really support merge).